service/radio_service.py

```diff
@@ -0,0 +1,13 @@
+def radio_action(adminVar):
```

Comment:
Can we keep this method in mainframe_support.py?

Reply:
yes

Reply:
this file is no longer needed, please delete it.